static/app/views/dashboardsV2/dashboard.tsx

```diff
@@ -363,6 +372,25 @@ class Dashboard extends Component<Props, State> {
     return <SortableWidget {...widgetProps} key={key} dragId={dragId} />;
   }
 
+  renderWidgets(widgets: Widget[]) {
+    const {layouts} = this.state;
+    let columnDepths = calculateColumnDepths(layouts[DESKTOP]);
+
+    return widgets.map((widget, index) => {
+      if (!defined(widget.layout)) {
+        const height = getWidgetHeight(widget.displayType);
+        const [nextPosition, nextColumnDepths] = getNextAvailablePosition(
+          columnDepths,
+          height
+        );
+        columnDepths = nextColumnDepths;
+        return this.renderWidget(widget, index, nextPosition);
+      }
+
+      return this.renderWidget(widget, index);
```

Comment:
I would invert this
```suggestion
      if (defined(widget.layout)) {
      	return this.renderWidget(widget, index);
      }
      
      const height = getWidgetHeight(widget.displayType);
      const [nextPosition, nextColumnDepths] = getNextAvailablePosition(
        columnDepths,
        height
      );
      columnDepths = nextColumnDepths;
      return this.renderWidget(widget, index, nextPosition);
```

less indenting ftw

Reply:
Great suggestion, thanks!